uxarray/grid/grid.py

```diff
@@ -2217,6 +2217,14 @@ def to_polycollection(
         """
         import cartopy.crs as ccrs
 
+        if kwargs.get("projection", False):
+            warn(
+                "Explicitly providing a projection when construction a PolyCollection has been deprecated. Please provide a projection when setting up"
```

Comment:
```suggestion
                "Explicitly providing a projection when constructing a PolyCollection has been deprecated. Please provide a projection when setting up"
```

Reply:
In addition to above, the last sentence reads like incomplete, how about something like "You provide a projection later, e.g. when setting up your plot with the PolyCollection."?